src/main/java/frc/robot/subsystems/ShooterSystem.java

```diff
@@ -84,6 +85,10 @@ public void rotatePID(double targetRPM){
         pid.setReference(targetRPM, CANSparkBase.ControlType.kVelocity);
     }
 
+    public boolean reachedRPM(double rpm){
```

Comment:
unused

Reply:
i didnt use this subsystem, should i still delete?